RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Release
+
+The kubernetes-base-addons repository is intended to support a minimal base set of Addons required to provide monitoring, logging, alerting, and backups for every version of Kubernetes actively supported by the upstream Kubernetes community.
+It is not intended to be tied to any specific version of Kubernetes installer (ie. Konvoy or Kommander), or Kubeaddons.
+Changes to Konvoy or Kommander to create a resource needed by an Addon should be avoided.
+
+- [Schedule](#schedule)
+- [Considerations](#considerations)
+  - [Kubernetes](#kubernetes)
+  - [Branches and Tags](#branches-and-tags)
+- [Process](#process)
+  - [Testing Release (Weekly, Thursday)](#testing-release-weekly-thursday)
+  - [Stable Release (Weekly, Wednesday)](#stable-release-weekly-wednesday)
+
+## Schedule
+
+Releases should be weekly on Wednesdays, or as needed to address CVEs.
+
+## Considerations
+
+### Kubernetes
+
+The Addons in this repo are the minimum base set of supported<sup>[1](#footnote1)</sup> Addons to be installed as a suite for any supported version of Kubernetes.
+
+Kubernetes support must be maintained for the latest general release of Kubernetes and the two prior minor releases.
+At the time of this writing, the latest Kubernetes release is 1.17.2.
+Support from this repo, at this time, should cover 1.15.x, 1.16.x, and 1.17.x.
+
+### Branches and Tags
+
+As much as possible, we will try to maintain a `master` branch that is compatible with all supported versions of Kubernetes.
+If there is a variation in the Kubernetes API which requires a _**breaking change**_ to the Addon, a _**branch**_ will be made for the prior Kubernetes versions.
+All future changes adopted into master will need to be back-ported to those branches.
+
+## Process
+
+### Testing Release (Weekly, Thursday)
+
+Each _**Thursday**_, this repository should be tagged for SOAK testing as follows:
+
+- Parse the PR logs for release notes and generate and commit a Changelog.md
+- One _**tag**_ is made for the each supported version of Kubernetes with an incremented release counter
+- If the current version of Kubernetes is `1.17.2`, and the last release was `stable-1.17-2`, the new SOAK tag will be `testing-1.17-3`.
```

Comment:
I think this can be confusing, I guess`stabel-1.17-2` should not be tied to k8s `1.17.2`? maybe just change k8s version to something else in order to clarify that release counter is unrelated to k8s patch releases:
```suggestion
- If the current version of Kubernetes is `1.17.7`, and the last release was `stable-1.17-2`, the new SOAK tag will be `testing-1.17-3`.
```

Reply:
+1 to julian. Since this is [written above](https://github.com/mesosphere/kubernetes-base-addons/pull/119/files#diff-781a53e648f3df8d16a08ec083b04bf4R26), I would suggest:
```suggestion
- If the current version of Kubernetes is `1.17.2`, and the last release was `stable-1.17-5`, the new SOAK tag will be `testing-1.17-6`.
```

Reply:
There should (I think k8s is stable enough that we can rely on this) never be a need to tie a tag/branch to a kubernetes patch release. The api should not be changing. So stable-`major`.`minor`-`dist`. I'll add text to back this up.